src/q4/opti-INMA1702/notes/cm/cm2-formes.tex

```diff
@@ -0,0 +1,273 @@
+\section{Formes standard et géométrique}
+
+\subsection{Problème d'affectation continu}
+
+	Intéressons-nous maintenant
+	à une formulation continue du problème d'affectation.
+
+	La seule chose qui change
+	par rapport au problème d'affectation discret (\sectionref{affectation})
+	est la contrainte
+
+	\[
+	x_{ij} \in \{0, 1\} \quad \forall i,j\,.
+	\]
+
+	Comme on se trouve maintenant dans une situation continue,
+	elle se transforme en
+
+	\[
+	x_{ij} \in \interval{0}{1} \quad \forall i,j\,.
+	\]
+
+\subsection{Forme géométrique}
+
+	La forme géométrique est une certaine façon
+	de représenter un problème d'optimisation linéaire.
+
+
```

Comment:
Also, what is the purpose ?

Reply:
Purpose of what?

Reply:
3 line breaks :smile:

Reply:
It's 3 line breaks, but only 2 blank lines. I was starting a new paragraph.

Reply:
I don't see any difference between 1 or 2 blank lines, do you? It's jjust making a break line + indentation in pdf.

![image](https://user-images.githubusercontent.com/25173236/44622034-1af8f080-a8b1-11e8-9aeb-fec16788ca82.png)

Reply:
Ah oui je me suis embrouillé apparemment...

Reply:
C'est pas grave, c'était juste pour ne pas mettre des lignes inutiles que je disais ça. Ce n'est pas si important. Sinon il y a `medskip` pour les espaces entre 2 lignes + alinéa. :wink: